### ISSUE TITLE
Displays irrelevant information

### ISSUE THREAD

**Issue by Drflash55** [dimotsai/agar-mini-map#42]
On the minimap, whenever you are playing Teams, it displays blobs on the minimapp that look like they should be in FFA only, and when you go towards those blobs, they disappear and don't show up on the actual map...

Only to reappear again when you walk away... This is a bug I've only seen recently in Teams mode, so when I went to FFA, those same blobs are still there...
I don't really know what's the cause of this...

![What is this?](http://i.imgur.com/xASchDk.png)

**Comment by dimotsai** [comment]
@Drflash55 Would you mind providing the reproduction steps and the environment? I tried but it seems work well.

**Comment by Drflash55** [comment]
@dimotsai 
I can't really re-produce it, because it just happens everywhere...
But, I can tell you what I've been doing...

I usually play on the Agar.io Official FFA Servers, and the Official Agar.io Team Servers, and they seem to appear out of nowhere...
It also happens when I play on the agariomods.com private servers as well...
I don't really know why it occurs, it just happens... And it didn't happen before at all when I first installed it... Only now.

**Comment by dimotsai** [comment]
@dimotsai Which browser do you use? What other plugins did you install for agario?

**Comment by Drflash55** [comment]
@dimotsai

1. I play using Google Chrome
2. I have the AposBot, but that is not the cause, because I always shut it off when I go to play Agariomods
3. It happens ALL the time...
4. I did check it, but it didn't show any errors in the JS console...

I have no idea why it happens.

**Comment by dimotsai** [comment]
Where did those invalid blobs come from? A previous connection? Did those blobs move?

I am not sure what happened on your minimap. But here are my suggestions:
1. Remove all plugins including Agariomods and try it again.
2. Go other browsers. Install the minimap and see whether this issue happens or not.

**Comment by dimotsai** [comment]
@Drflash55 It seems like that the minimap is not compatible with A1ien's Agar Mod. The minimap is fine in FFA mode but it does not work properly in Team mode.

**Comment by Drflash55** [comment]
@dimotsai The blobs seem to be from another FFA map that I'm not even playing on, as they always seem to be rendering out of my actual view. For example, something in FFA is happening in the middle of the map, while I'm at the bottom left...
That's not really supposed to be happening, and when I head over, those blobs disappear...

**Comment by Drflash55** [comment]
@dimotsai 
Here I am, 2 months later, asking you if you found out the solution for this.

I haven't used this in a while, but if I was going to, I would only use it if this bug was fixed.

**Comment by dimotsai** [comment]
Yes, it should be no problem as long as you remove other plugins. I am
quite sure that the issue was caused by some incompatible plugins.

Drflash55 <notifications@github.com> 於 2015年11月6日週五 下午8:30 寫道：

**Comment by Drflash55** [comment]
@dimotsai 
I will test out this script without other plugins interfering with this soon...
